### PR TITLE
{data}[intel/2018a] PostgreSQL v10.2

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-10.2-intel-2018a-Python-2.7.14.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    # it's nice to have an up to date openssl for security reasons
+    # ('OpenSSL', '1.0.2h'),
+]
+
+osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
+
+builddependencies = [
+    ('Bison', '3.0.4'),
+    ('flex', '2.6.4'),
+    ('Perl', '5.26.0'),
```
Comment (boegel) [inline easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-10.2-intel-2018a-Python-2.7.14.eb]: @SimonPinches Please bump to `5.26.1` (required easyconfig is already available in `develop`).

Comment (SimonPinches) [reply easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-10.2-intel-2018a-Python-2.7.14.eb]: Done.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2428.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/829dc53a14b1530354752935f79722ab for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar02.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/03067289afd0ba466eb5fe050f1c73e6 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @SimonPinches!

**Pull request merged**